caom2pipe/execute_composable.py

```diff
@@ -1432,11 +764,35 @@ def __init__(
             self._store_transfer.observable = self.observable
         self._cadc_client = cadc_client
         self._caom_client = caom_client
-        self._cred_param = self._define_cred_param()
+        self._metadata_reader = metadata_reader
         self._log_h = None
         self._logger = logging.getLogger(self.__class__.__name__)
         self._logger.setLevel(config.logging_level)
 
+    def _clean_up_workspace(self, obs_id):
+        """Remove a directory and all its contents. Only do this if there
+        is not a 'SCRAPE' task type, since the point of scraping is to
+        be able to look at the pipeline execution artefacts once the
+        processing is done.
+        """
+        working_dir = os.path.join(self.config.working_directory, obs_id)
+        self._logger.debug(
```

Comment:
Shouldn't this message be moved after l785 when the cleanup actually takes place?